src/wp-admin/includes/media.php

```diff
@@ -3260,7 +3260,7 @@ function edit_form_image_editor( $post ) {
 		'textarea_name' => 'content',
 		'textarea_rows' => 5,
 		'media_buttons' => false,
-		'tinymce'       => false,
+		'tinymce'       => apply_filters( 'activate_tinymce_for_media_description', false ),
```

Comment:
```suggestion
		/**
		 * Filters the media description for tinymce.
		 *
		 * @since 6.5
		 *
		 * @param bool Activate media description for tinymce. Default false.
		 */
		'tinymce'       => apply_filters( 'activate_tinymce_for_media_description', false ),
```

Add missing docs for filter. Please update suggestion if it need any update.